src/main/webapp/courseLoader.js

```diff
@@ -259,13 +261,39 @@
     return prevTermData.map((data) => data.properties.term);
   }
 
-  function makeGradeChart() {
+  function makeGradeChart(termDataObject) {
+    // prettier-ignore
+    const gradeMapper = {
+      "A+": 97.5,
+      "A": 95,
+      "A-": 92.5,
+      "B+": 87.5,
+      "B": 85,
+      "B-": 82.5,
+      "C+": 77.5,
+      "C": 75,
+      "C-": 72.5,
+      "D+": 67.5,
+      "D": 65,
+      "D-": 62.5,
+      "F": 55,
+    };
+
+    const numericalGrade = [];
+    for (let grade of termDataObject.gradesList) {
+      if (grade != "I") {
```

Comment:
Are we guaranteed gradesList will just be either things in gradeMapper or "I"?
If there's any weird data here it will still get pushed.

Reply:
Yes we are guaranteed this because the grade is selected via drop-down so the user is limited in what they input